maps/saveData.lua

```diff
@@ -0,0 +1 @@
+data = {credits = 0, time = '2:33:44', lastMap = 1}
```

Comment:
- This should be gitignored if it is custom user data.
- What about prefixing the file with `return` so it returns a value instead of having the side effect of setting a global variable `data = `.
- Also why save in the map folder? Maybe we want to store non-map information.

Reply:
- Good point
- Good point
- I wasn't sure where to put it, and the maps folder just felt right. I can just drop it in the main folder if you'd like.

Reply:
My suggestion would be a dedicated save directory. Player data and map times could all be located in it

Reply:
Use the save directory that LOVE sets up. https://love2d.org/wiki/love.filesystem
Since the game could be ran from an executable (once distributed) the save directory is ideal because it's a separate folder in the OS's app_data directory.